extensions/2.0/Vendor/DRACO_animation_compression/README.md

```diff
@@ -0,0 +1,236 @@
+# Draco_animation_compression
+
+## Contributors
+
+* Fan Zhang, Google, <mailto:zhafang@google.com>
+* Michael Hemmer, <mailto:hemmer@google.com>
+* Ondrej Stava, Google, <mailto:ostava@google.com>
+* Frank Galligan, Google, <mailto:fgalligan@google.com>
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines a schema to use [Draco compression library (non-normative)](https://github.com/google/draco) to compress `animations` in glTF format. This allows glTF to support streaming compressed animation data instead of the raw data.
+
+The [conformance](#conformance) section specifies what an implementation must do when encountering this extension, and how the extension interacts with the attributes defined in the base specification.
+
+## glTF Schema Updates
+
+If a Draco compressed version of one or more `animations` is provided then `Draco_animation_compression` must be added to `extensionsUsed`.
+
+```javascript
+"extensionsUsed" : [
+    "Draco_animation_compression"
+]
+```
+
+If the uncompressed version of one or more `animations` are not provided, then `Draco_animation_compression` must be added to `extensionsRequired`.
+```javascript
+"extensionsRequired" : [
+    "Draco_animation_compression"
+]
+```
+
+If the root object of a glTF asset contains `extensions` property and the `extensions` property defines its `Draco_animation_compression` property, then the Draco animation compression must be used. If there are any compressed animations then the `Draco_animation_compression` must define the `compressedAnimations` property. The `compressedAnimations` property consists of an array of compressed animation objects. Each element in `compressedAnimations` must have the following properties:
+
+#### input
+The `input` property defines a single `input` of a compressed animation. The
+`input` corresponds to a `samplers` element's `input`.
```

Comment:
I'm not sure I understand the meaning of "corresponds to" here — what is this an index of? An accessor index, that is also referenced by a sampler's input? Or an index of the sampler whose input is overridden?

Reply:
So this is an accessor index, that is also referenced by the sampler's input.

I guess we don't really need to say anything about the sampler's input here.

How about:
The `input` property defines a single `input` of a compressed animation. The
`input` references the accessor to use for the input data.

Reply:
That seems clear enough then, yes. I think we should also specify that the accessor cannot be referenced by anything other than a sampler input/output.

Can a compressed `input` be used in a sampler alongside an uncompressed `output`, and vice versa? I would lean toward disallowing both.